### PR TITLE
Refresh network-dell-nseries-snmp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-dell-nseries-snmp

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-dell-nseries-snmp.md [hunk 1]
```diff
@@ -5,94 +5,126 @@ title: Dell N-series SNMP
...
 
-| Metric name           | Description        | Unit  |
-| :-------------------- | :----------------- | :---- |
-| system.uptime.seconds | System uptime      | s     |
+| Nom                   | Unité |
+|:----------------------|-------|
+| system.uptime.seconds | s     |
 
 </TabItem>
 </Tabs>
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-dell-nseries-snmp.md:130]: prérequis standard SNMP

File: pp/integrations/plugin-packs/procedures/network-dell-nseries-snmp.md [hunk 2]
```diff
@@ -5,94 +5,125 @@ title: Dell N-series SNMP
...
 <Tabs groupId="sync">
-<TabItem value="Service" label="Service">
+<TabItem value="Net-Dell-Nseries-SNMP-custom" label="Net-Dell-Nseries-SNMP-custom">
 
-| Rule name                            | Description                                         |
-|:-------------------------------------|:----------------------------------------------------|
-| Net-Dell-Nseries-SNMP-Interface-Name | Discover network interfaces and monitor utilization |
+| Service Alias | Service Template                         | Service Description                                                                                           |
+|:--------------|:-----------------------------------------|:--------------------------------------------------------------------------------------------------------------|
+| Cpu           | Net-Dell-Nseries-Cpu-SNMP-custom         | Check the rate of utilization of CPU for the machine. This check can give the average utilization rate of CPU |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-dell-nseries-snmp.md:29]:
```suggestion
| Cpu           | Net-Dell-Nseries-Cpu-SNMP-custom         | Check the rate of CPU utilization for the machine. This check can give the average utilization rate of the CPU |
```

File: pp/integrations/plugin-packs/procedures/network-dell-nseries-snmp.md [hunk 3]
```diff
@@ -5,94 +5,125 @@ title: Dell N-series SNMP
...
 </TabItem>
 </Tabs>
 
+### Discovery rules
+
+#### Host discovery
+
+| Rule name       | Description                                                                                                                                                                                                                                  |
+|:----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Découvre les ressources via un scan réseau SNMP. Installez le connecteur [Generic SNMP](./applications-protocol-snmp.md) pour obtenir la règle de découverte et créez un modificateur pour le modèle d'hôte **Net-Dell-Nseries-SNMP-custom** |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-dell-nseries-snmp.md:57]: FR in EN

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/network-dell-nseries-snmp.md [hunk 4]
```diff
@@ -5,94 +5,125 @@ title: Dell N-series SNMP
...
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-dell-nseries-snmp.md:57]:
```suggestion
| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **Net-Dell-Nseries-SNMP-custom** host template |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-dell-nseries-snmp.md [hunk 5]
```diff
@@ -5,94 +5,126 @@ title: Dell N-series SNMP
...
 
+### Règles de découverte
+
+#### Découverte d'hôtes
+
+| Nom de la règle | Description                                                                                                                                                      |
+|:----------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-dell-nseries-snmp.md:58]:
```suggestion
| SNMP Agents     | Découvre les ressources via un scan réseau SNMP. Installez le connecteur [Generic SNMP](./applications-protocol-snmp.md) pour obtenir la règle de découverte et créez un modificateur pour le modèle d'hôte **Net-Dell-Nseries-SNMP-custom** |
```

**Pull request merged**